README.md

```diff
@@ -83,19 +98,24 @@ var obj = {
 }
 ```
 
--- YOUR ANSWER HERE --
+- I don't think this is valid JS because the contents of the object are not setup as key:value pairs
+
 
 ## What does the following code output to the console?
 
 ```js
 var i = 0
-for ( console.log('Bob'); (console.log('Curly') === undefined); console('Mo') ) { 
+for ( console.log('Bob'); (console.log('Curly') === undefined); console('Mo') ) {
   console.log(++i)
   if (i >= 2)
     break
 }
 console.log(i)
 ```
 
--- YOUR ANSWER HERE --
-
+Bob
+Curly
+Mo
+1
+2
+2
```

Comment:
The correct output is: 

Bob
Curly
1
Mo
Curly
2
2

Play with the for loop syntax like Carlos did in class